clinica/pipelines/deeplearning_prepare_data/deeplearning_prepare_data_cli.py

```diff
@@ -89,7 +92,9 @@
         "`segm-whitematter_probability.nii.gz`"
     ),
 )
-@cli_param.option_group.standard_options
+@optgroup.group(
+    "Common pipeline options", help="Options common to all Clinica pipelines"
+)
```

Comment:
What about defining this group in [option_group.py](https://github.com/aramis-lab/clinica/blob/dev/clinica/pipelines/cli_param/option_group.py)? This would avoid copy/paste in other `*.cli.py` files

Reply:
Indeed much better ! thanks.